daemon/cmd/policy_test.go

```diff
@@ -377,7 +377,7 @@ func (ds *DaemonSuite) TestUpdateConsumerMap(c *C) {
 
 	expectedNetworkPolicy = &cilium.NetworkPolicy{
 		Name:             ProdIPv4Addr.String(),
-		Policy:           uint64(prodBarSecLblsCtx.ID),
+		EndpointId:       uint64(eProdBar.ID),
```

Comment:
Interesting, so the policy API between cilium-agent and the proxy has changed?

I think this is fine given that we always ship Envoy alongside Cilium. So we shouldn't need to deal with compatibility issues of mismatched versions attempting to use different field names from what the other binary expects.